Wrappers/Python/ccpi/viewer/viewerLinker.py

```diff
@@ -301,9 +301,9 @@ def __call__(self, interactor, event):
                 shouldPassEvent = False
             else:
                 # Set current window/level
-                window = self.sourceViewer.getColourWindow()
-                level = self.sourceViewer.getColourLevel()
-                self.targetVtkViewer.setColourWindowLevel(window, level)
+                window = self.sourceViewer.getSliceColourWindow()
+                level = self.sourceViewer.getSliceColourLevel()
+                self.targetVtkViewer.setSliceColourWindowLevel(window, level)
```

Comment:
I wonder if we want the british spelling for `colour` or we could be happy with the more generally accepted (in the IT world) `color`.
This is a general comment, rather than a request of changing of this specific method names, unless `getSliceColourWindow/Level` are new methods.